src/modules/simulation/gz_bridge/GZBridge.cpp

```diff
@@ -196,11 +196,20 @@ int GZBridge::init()
 		return PX4_ERROR;
 	}
 
+
+	// Distance Sensor(AFBRS50): optional
+	std::string lidar_sensor = "/world/" + _world_name + "/model/" + _model_name +
+				   "/link/BroadcomSensor/sensor/broadcomafbrs50/scan";
```

Comment:
Why is this specific to a broadcom topic?

Reply:
That is the model of the rangefinder. It could be renamed, but would have to differentiate from the other lidar that is used for obstacle distance messages and not as a range aid.